app/models/event.rb

```diff
@@ -240,5 +241,12 @@ def user_cannot_be_inactive
 
     errors.add(:base, "You need an active subscription to capture events with your live API key")
   end
+
+  def property_name_cannot_be_excluded
+    excluded_values = %w(All all)
```

Comment:
Better var name: `excluded_property_names`.

Also, Rubocop should tag this, but `%w[All all]` is the preferred syntax I think?